zendesksupport/openapi/file.go

```diff
@@ -0,0 +1,16 @@
+package openapi
+
+import (
+	_ "embed"
+
+	"github.com/amp-labs/connectors/tools/fileconv/api3"
+)
+
+var (
+	// Static file containing openapi spec.
+	//
+	//go:embed support-api.yaml
+	apiFile []byte
+
+	FileManager = api3.NewOpenapiFileManager(apiFile) // nolint:gochecknoglobals
```

Comment:
`support-api.yaml` is loaded into FileManager.

The script uses this FileManager along with the OpenAPI tool to produce `schemas.json`.